### PR TITLE
Add Notion.app v0.3.0

### PR DESCRIPTION
**Pull request by albertogalca**
After making all changes to the cask:

- [X] `brew cask audit --download notionso` is error-free.
- [X] `brew cask style --fix notionso` reports no offenses.
- [X] The commit message includes the cask’s name and version.
- [X] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [X] Named the cask according to the [token reference].
- [X] `brew cask install notionso` worked successfully.
- [X] `brew cask uninstall notionso` worked successfully.
- [X] Checked there are no [open pull requests] for the same cask.
- [X] Checked the cask was not [already refused].
- [X] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/notionso.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'notionso' do
+  version '0.3.0'
+  sha256 :no_check
```
Comment (vitorgalvao) [inline Casks/notionso.rb]: `sha256` also needs to be set. You can find what it is by running `shasum -a 256` on the downloaded file.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by core-code** [comment]
i am not sure what the point of this was - we already have this cask under its proper name 'notion'

**Comment by vitorgalvao** [comment]
@core-code That was @albertogallego’s failure to follow the token reference, and my failure to notice it wasn’t properly named. These don’t typically trip me up, but being only two extra letters with similar shapes, my brain must’ve ignored it.

Removing it now. Thank you, as always.

**Comment by core-code** [comment]
no problem, no harm done, you are welcome

**Pull request merged**